vmdb/app/models/vm_or_template/operations/smart_state.rb

```diff
@@ -0,0 +1,9 @@
+class VmOrTemplate
+  module Operations
+    module SmartState
+      def validate_scan
+        {:available => true, :message => nil}
```

Comment:
@Fryguy Yes, we've been trying to push these types of decisions back to the models as much as possible.